src/onboarding/containers/LoginPage.tsx

```diff
@@ -47,6 +49,17 @@ export const LoginPage: FC = () => {
     }
     return null
   }
+  else {
+    if (CLOUD) {
+      const url = new URL(
```

Comment:
we would add one more check here to make sure this is working for staging / tools as well.